vnext/ReactWindowsCore/Modules/AsyncStorageModuleWin32.h

```diff
@@ -22,9 +23,33 @@ class AsyncStorageModuleWin32 : public facebook::xplat::module::CxxModule {
   std::map<std::string, dynamic> getConstants() override;
   std::vector<facebook::xplat::module::CxxModule::Method> getMethods() override;
 
-  using ExecCallback = int(SQLITE_CALLBACK *)(void *, int, char **, char **);
-
  private:
+  class DBTask {
+   public:
+    enum class Type { multiGet, multiSet, multiRemove, clear, getAllKeys };
+    DBTask(Type type, folly::dynamic &&args, Callback &&callback)
+        : m_type{type}, m_args{std::move(args)}, m_callback{std::move(callback)} {}
+    DBTask(const DBTask &) = delete;
+    DBTask(DBTask &&) = default;
+    DBTask &operator=(const DBTask &) = delete;
+    DBTask &operator=(DBTask &&) = default;
+    void operator()(sqlite3 *db);
+
+   private:
+    Type m_type;
+    folly::dynamic m_args;
+    Callback m_callback;
+
+    void multiGet(sqlite3 *db);
+    void multiSet(sqlite3 *db);
+    void multiRemove(sqlite3 *db);
+    void clear(sqlite3 *db);
+    void getAllKeys(sqlite3 *db);
+  };
+  winrt::slim_mutex m_lock;
+  winrt::slim_condition_variable m_cv;
+  winrt::Windows::Foundation::IAsyncAction m_action;
```

Comment:
>m_action [](start = 43, length = 8)

Please pass {nullptr} here to ensure it to be initialized with nullptr instead of creating a new value.

Reply:
You're confusing how C++/WinRT interfaces and classes work, but it's harmless to add `{nullptr}` here to help avoid confusing others, sure.